homeassistant/components/webostv/media_player.py

```diff
@@ -332,11 +345,22 @@ async def async_turn_on(self):
     async def async_volume_up(self):
         """Volume up the media player."""
         await self._client.volume_up()
+        await self._sleep_between_consecutive_volume_steps()
 
     @cmd
     async def async_volume_down(self):
         """Volume down media player."""
         await self._client.volume_down()
+        await self._sleep_between_consecutive_volume_steps()
```

Comment:
Shouldn't the library handle this guard instead?

Reply:
You're thinking this functionality belongs in aiopylgtv instead? Hmm, yes I guess that kinda makes sense. I'll look into if I can make it handle it instead

Reply:
@MartinHjelmare I've open a PR against aiopylgtv instead as it can handle this instead, which is a better idea than to handle it here. So I'm closing this PR. Thanks for your help :)